azurelinuxagent/common/cgroups.py

```diff
@@ -556,16 +564,21 @@ def setup(suppress_process_add=False):
             try:
                 CGroups._osutil.mount_cgroups()
                 if not suppress_process_add:
+                    # Creates /sys/fs/cgroup/{cpu,memory}/WALinuxAgent wrapper cgroup
                     CGroups._setup_wrapper_groups()
                     pid = int(os.getpid())
                     if not CGroups.is_systemd_manager():
```

Comment:
Can we flip this `if`? Majority of the cases are `CGroups.is_systemd_manager() == true` cases, and is better on the eyes.

Reply:
Changed.